components/button/test/button.test.js

```diff
@@ -47,16 +47,20 @@ describe('button', () => {
 			let submitted = false;
 			const addedElements = addElement(
 				textToDomToParent(
-					`<form onsubmit="return false" name="testForm" id="testForm"><${COMPONENT_NAME}>Button Text</${COMPONENT_NAME}></form>`
```

Comment:
Why did you change an existing test?

Reply:
that's the default submission test.. there is no significance in testing submission without value & name alongside another test without them.